src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/RegExpSpec/help-showUpstream.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Will show upstream jobs for the first job.
```

Comment:
@tommysdk i had few critical questions in terms of time when i searched for answers.